### PR TITLE
Allow multiple certs per entry

### PR DESCRIPTION
**Pull request by dajudge**
This is a first attempt at fixing #142 

It's parsing my k3s kubeconfig successfully.

### CODE REVIEW

File: test/yaml-multi-certs.conf [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
...STRNVEl5TXpFek1qY3pORm93RlRFVE1CRUdBMVVFCkF4TUthM1ZpWlhKdVpYUmxjekNDQVNJd0RRWUpLb1pJaHZjTkFRRUJCUUFEZ2dFUEFEQ0NBUW9DZ2dFQkFQS1gKRkozSlFQMWVLVS9rMy9XazB5YUtIRC9PZjVxV2syUDMxVXVpb202SjlSUzZNbE54V0RGQmVycXpRaXNDSDZqKwphK3MzeGkzZ2ZRdWVWcXpUK1B6VGk1aDNJK0dwVGdqOWZ1OHpZeTFtRGIrdG95dGpIdXM5RTZFUXhmcXRIZFR4CjNrZERKWmVVYkJlR01JeVJHcUx5c09PUFZ1cHdRYjcwMWVnUXJYd21sTG9yMmZpTFBHQWJ5R0FCMHJjRWJDWkQKaHYzMDVtNlNFdjdZT0pONG9WTDFMallGTkV2WkJVcUlMUi9LbkN4Y0pzcFdralBJT21CL1VYSUxsTGNoSURobApDYVplaW9lYjdaUHJjYlp6bmpiSE9jaVlpSVNJNTBKKzYxeXd4ejFnMEpQMmlmdGJmYzBML1BZVDZIOXgxL3hxCjQrVzVBSldWZVZ5UUllaGZwNmNDQXdFQUFhTWpNQ0V3RGdZRFZSMFBBUUgvQkFRREFnS2tNQThHQTFVZEV3RUIKL3dRRk1BTUJBZjh3RFFZSktvWklodmNOQVFFTEJRQURnZ0VCQUxMOEI1bGpOSHJwajRrSTEvRkFKZmpoOEFrdQpPN2kySEtHZEdnRnJtR0x5R1pFM1JERENrcjZQWlF2VVhCSGZPMzZ0NzdLbXBhbUU5NSthMmQ5a3piV3dNSXc1ClIwRGUrL3l4ckNrcFFxNWZraUlBb1UwQitrWUZ2Z2ZvWWUyb1FSMFVJc3VJY0lxU29Ca2dSOUZBdjN0MEdXbmIKVllrVUo5OXFCbkdPT05TVEtzT0hvb1F6aStWc0k3QzdLUG9lMzM2QlpWeVZNNEdLV0lKeU1JN08rKy9ScVUrZgprQXFCdXNwcG5kbFdERmxqVnA2TnJ2SGJmNFppbENsSW5RU3Q0eDc3SHpXQVJqajdjdk1vTWdISkRVdklBL01HCkhkQVhiMnY3dWlmOEtPMUorRnNrS1B2ZUN1Y0J4OEhQOUZOUjE2dlJINWRkUE8rTERTTEIxU1lndStjPQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
```
Comment (dajudge) [inline test/yaml-multi-certs.conf:4]: 2 certs now

File: test/yaml-multi-certs.conf [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
     server: https://127.0.0.1:6443
+  name: default-cluster
```
Comment (dajudge) [inline test/yaml-multi-certs.conf:6]: one user with 2 certs

**Comment by arcln** [comment]
Thank you, everything looks good. Please mark this PR as non-draft if you're done with it. We'll merge after that.

**Comment by dajudge** [comment]
🎉🎉🎉

**Pull request merged**